sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/StringConcatenationInLoop.cs

```diff
@@ -71,5 +75,9 @@ public sealed class StringConcatenationInLoop
         protected override ImmutableArray<SyntaxKind> CompoundAssignmentKinds => compoundAssignmentKinds;
 
         protected sealed override Helpers.GeneratedCodeRecognizer GeneratedCodeRecognizer => Helpers.CSharp.GeneratedCodeRecognizer.Instance;
+
+        protected override bool IsAddExpression(BinaryExpressionSyntax rightExpression) =>
+            rightExpression != null &&
+            rightExpression.OperatorToken.IsAnyKind(AddOperators);

```

Comment:
Can `OperatorToken` be null? If so is `IsAnyKind` handling null?

Reply:
Cannot be, it's a struct.